app/earthquakes.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace App;
+use Illuminate\Database\Eloquent\Model;
+class earthquakes extends Model
```

Comment:
Please start your Model name with a capital letter. We always start our class names with a capital letter. I said this in class.